src/components/ProductCardContainer/styles.js

```diff
@@ -0,0 +1,104 @@
+export const productCardStyles = {
+  display: "flex",
+  overflow: "hidden",
+  width: 580,
+  height: 200,
+  alignItems: "center",
+  justifyContent: "space-between",
+  borderRadius: 6,
+  border: "1px solid rgba(53, 184, 190, 0.15)",
+  background: "#FFF",
+};
+
+export const namePriceContainer = {
+  display: "flex",
+  flexDirection: "row",
+  justifyContent: "space-between",
+  width: 378,
+  height: 40,
+};
+
+export const productPrice = {
+  margin: 0,
+  padding: 0,
+  color: "#35B8BE",
+};
+
+export const productName = {
+  margin: 0,
+  color: "#08090A",
+  fontFamily: "Inter",
+  fontSize: 18,
+  fontStyle: "normal",
+  fontWeight: 400,
+  letterSpacing: 0.6,
+};
+
+export const productDescribtion = {
+  display: "flex",
+  margin: 0,
+  padding: 0,
+  minWidth: 351,
+  color: "#546285",
+  fontFamily: "Inter",
+  fontSize: 16,
+  fontStyle: "normal",
```

Comment:
It is a default font-style, so, we don't have to type it manually if there is no necessity